### DISCUSSION TITLE
feature request: drag and drop files and folders from anywhere  to 'tree view panel'

### DISCUSSION THREAD

**Discussion by nutpantz** [doublecmd/doublecmd#1140] [Suggestions]
while i may have folders open anywhere, i usually can see a overview of the system drives and folders in the "tree view panel" it would nice to be able to drag and drop anything from the other panels to the "tree view panel" drive/folder and then have the dragged files or folders copied or moved there.
(which i was used to with directory opus)

**Comment by vertigo220** [comment]
I just assumed you could until I ran across this issue searching before posting other requests for improvements to the tree view, which now seem a bit pointless since the whole tree view is borderline useless without this functionality. I'm surprised it can't already do this. And it's been > 4 years since this was requested, so doesn't seem like something that will ever be done.
